src/output/plugins/SndioOutputPlugin.cxx

```diff
@@ -44,30 +45,21 @@ static constexpr unsigned MPD_SNDIO_BUFFER_TIME_MS = 250;
 
 static constexpr Domain sndio_output_domain("sndio_output");
 
-class SndioOutput final : AudioOutput {
```

Comment:
This removal is obscure and hard to review, because it moves code and modifies it at the same time. Please make the move a separate commit.